app/containers/Article/index.js

```diff
@@ -46,18 +54,18 @@ export class Article extends React.PureComponent { // eslint-disable-line react/
         />
         <hr />
         <div dangerouslySetInnerHTML={{ __html: body }} />
-        <Comments fetching={fetching} comments={this.props.comments} />
+        <Comments fetching={fetching} comments={data} />
```

Comment:
make `error`, `fetching` and `comments` required in propTypes for `<Comments/>` component